src/docs/operator-guides/edit-seo-and-social-tags/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to edit SEO & social tags in Pages
+slug: how-to-edit-seo-and-social-tags-in-pages
+updated: 2023-02-08
+category: operator-guides
+ingress: Change SEO and social tags in Pages
+published: true
+---
+
+Meta tags are a fundamental part of any static content page. They
```

Comment:
Vesa L has often pointed out to me that these are technically not "meta tags" and should not be called that. That's why we don't use that word in the Console copy.

I would have page title as something like this: "Change metadata for SEO and Social media".

In the opening paragraph I would say: "Every page has metadata that determines how search engines and social media platforms will display it. Google, for example, uses the title and description you set to your page to
populate their search results page.